Tests/ScenarioTests.swift

```diff
@@ -32,6 +33,8 @@ class ScenarioTests: XCTestCase {
                         context: nil,
                         api: api,
                         queue: DispatchQueue.main)
+        
+        redirectUrl = try XCTUnwrap(URL(string: "com.preview.example:///callback?interaction_code=qwe4xJasF897EbEKL0LLbNUI-QwXZa8YOkY8QkWUlpXxU&state=46C40AC5-9DFW-55CE-B291-31A4854F4164#_=_"))
```

Comment:
What is this variable used for? I don't see it accessed anywhere.

Reply:
It's used in `testScenario5`.

Reply:
Refactored.